src/main/java/app/App.java

```diff
@@ -20,6 +19,7 @@
     @Getter private StartMenu startMenu;
     @Getter private Stage stage;
     private Simulation simulation;
+    //private Timeline animation;
```

Comment:
Is this variable being used?

Reply:
ah, sorry, forgot to delete from wrong attempt